docs/mkdocs_macros.py

```diff
@@ -2,14 +2,32 @@
 
 from helm.benchmark.run_expander import RUN_EXPANDERS
 from helm.proxy.models import ALL_MODELS
+import yaml
 
 
 def define_env(env):
     @env.macro
     def models_by_organization():
+        with open("src/helm/benchmark/static/schema.yaml", "r") as f:
+            schema = yaml.safe_load(f)
+        models = list(schema["models"])
         result = defaultdict(list)
-        for model in ALL_MODELS:
-            result[model.creator_organization].append(model)
+
+        # Create dict name -> madel_object (ALL_MODELS)
+        name_to_model_object = {}
+        for model_object in ALL_MODELS:
+            name_to_model_object[model_object.name] = model_object
+
+        for model in models:
+            # Find the model in ALL_MODELS in an efficient way
+            model_object = name_to_model_object.get(model["name"], None)
+            if model_object is not None:
```

Comment:
In general, it'd be good to avoid using raw hash lookups and use dataclasses instead.